echo_client.py

```diff
@@ -14,5 +14,5 @@
 sock.sendall('Some message'.encode())
 data = sock.recv(1024)
 sock.close()
-print ("Received", repr(data))
+print ("Received", repr(data.decode()))
```

Comment:
Good to see that message is being decoded upon receiving by client.
I would remove also "repr" part and maybe set print to something like:
```suggestion
print ("Received: {}".format(data.decode()))
```